lib/clusters/metering.js

```diff
@@ -9,6 +9,7 @@ const ATTRIBUTES = {
   currentMaxDemandDelivered: { id: 2, type: ZCLDataTypes.uint48 },
   currentMaxDemandReceived: { id: 3, type: ZCLDataTypes.uint48 },
   powerFactor: { id: 6, type: ZCLDataTypes.int8 },
+  meterStatus: { id: 512, type: ZCLDataTypes.map8('checkMeter', 'lowBattery', 'tamperDetect', 'powerFailure', 'powerQuality', 'leakDetect', 'leakDetect', 'serviceDisconnectOpen') },
```

Comment:
I think you added `leakDetect` twice, can not find that in the spec.

Reply:
you're right should be 
`meterStatus: { id: 512, type: ZCLDataTypes.map8('checkMeter', 'lowBattery', 'tamperDetect', 'powerFailure', 'powerQuality', 'leakDetect', 'serviceDisconnectOpen', 'reserved') },`